cryptofeed/exchanges/bybit.py

```diff
@@ -475,14 +475,14 @@ async def _order(self, msg: dict, timestamp: float):
                 self.id,
                 self.exchange_symbol_to_std_symbol(data['symbol']),
                 data["order_id"],
-                BUY if msg["side"] == 'Buy' else SELL,
+                BUY if data["side"] == 'Buy' else SELL,
                 order_status[data["order_status"]],
                 LIMIT if data['order_type'] == 'Limit' else MARKET,
                 Decimal(data['price']),
-                Decimal(data['cumQuantity']),
-                Decimal(data['qty']) - Decimal(data['cumQuantity']),
-                self.timestamp_normalize(data["updateTime"]),
-                raw=data
+                Decimal(data['cum_exec_qty']),
+                Decimal(data['qty']) - Decimal(data['cum_exec_qty']),
+                self.timestamp_normalize(data["update_time"]),
```

Comment:
update_time is only available in USDT perpetuals, the others look like they support the `timestamp` field. Can you change this to use `update_time` if it exists in the json, otherwise use `timestamp`?

Reply:
Hey, I've added both timestamp and 'O' from spot market also

Reply:
ah ok, great!